packages/components/src/mobile/link-picker/index.native.js

```diff
@@ -73,20 +73,11 @@ export const LinkPicker = ( {
 		pickLink( directEntry );
 	};
 
-	const clear = () => {
-		setValue( { value: '', clipboardUrl } );
```

Comment:
I noticed that we would stop setting the `clipboardUrl` state value when clearing the text input if we remove this function. Not sure about potential side effects but probably would be worth investigating it.

Reply:
This is a very good point, and an edge case that I was considering how to address. I've updated BottomSheet.Cell to use an onClear handler as a prop, so that the parent component (like this one) can set the TextInput value directly, which handles this case. (More info in my comment below.)

Reply:
Great, thanks for addressing the issue, I'll check the comments.